Lombiq.UIKit.Widgets/Constants/ContentTypes.cs

```diff
@@ -0,0 +1,9 @@
+namespace Lombiq.UIKit.Widgets.Constants;
+
+public static class ContentTypes
+{
+    public const string SlidePart = nameof(SlidePart);
```

Comment:
Part names are not content types. You can keep reference them as `nameof(PartName)` so you don't need consts for these.